scripts/write-configs.js

```diff
@@ -0,0 +1,111 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
```

Comment:
Is there a reason to do that over `import xxx from '';`?

Reply:
`import xxx from '';` doesn't work in a standard js file with node, I would have to make it an .mjs file for this to work. But I think I didn't get the importing of the .ts file on line 12 working that way. I only got that working with `require()` which requires it to be a .js file.

Reply:
Or make the package a `"type": "module"`.

Reply:
This doesn't solve the problem with importing the .ts file on line 12 I think...at least I didn't figure out how to do it.

Reply:
And I would have to create a dedicated package for this because the kangaroo package cannot be a `"type": "module"` as the electron part of things would not build in that case.